Compilation issues

While compiling on Ubuntu 14.044.3, gcc version 4.8.4, I got the following error:
<pre>$ make
cc -std=c99    fastbin_dup.c   -o fastbin_dup
cc -std=c99    fastbin_dup_into_stack.c   -o fastbin_dup_into_stack
cc -std=c99    unsafe_unlink.c   -o unsafe_unlink
cc -std=c99    house_of_spirit.c   -o house_of_spirit
cc -std=c99    poison_null_byte.c   -o poison_null_byte
cc -std=c99    malloc_playground.c   -o malloc_playground
malloc_playground.c: In function ‘main’:
malloc_playground.c:27:3: warning: ‘gets’ is deprecated (declared at /usr/include/stdio.h:638) [-Wdeprecated-declarations]
   gets(buffer);
   ^
/tmp/ccquQQGB.o: In function `main':
malloc_playground.c:(.text+0xc2): warning: the `gets' function is dangerous and should not be used.</pre>

--- a/malloc_playground.c
+++ b/malloc_playground.c
@@ -24,7 +24,7 @@ int main(int argc, char ** argv) {
 	char buffer[1000];
 	while (1) {
 		printf("> ");
-		gets(buffer);
+		fgets(buffer, sizeof(buffer), stdin);
 		char cmd[1000];
 		intptr_t arg1, arg2;
 		int num = sscanf(buffer, "%s %"SCNiPTR" %"SCNiPTR, cmd, &arg1, &arg2);

